docs/docker.md

```diff
@@ -2,14 +2,21 @@
 
 The Docker image of Trino Gateway is designed for the following use cases:
 
-* Manual usage in front of Trino clusters by mounting in a configuration file
-* Automated usage with an orchestration system like Kubernetes to simplify
-  deployment
+* Manual usage in front of Trino clusters by mounting in a configuration file.
```

Comment:
"mounting in a configuration file" - I think this means mounting a configuration file to the container, since you can't mount an image in a file :). The wording may be confusing to some